com.unity.render-pipelines.universal/Editor/Camera/UniversalRenderPipelineCameraEditor.cs

```diff
@@ -499,23 +666,39 @@ void DrawStackSettings()
                 m_LayerList.DoLayoutList();
                 m_SerializedCamera.serializedObject.ApplyModifiedProperties();
 
-                if (errorCameras.Any())
+                EditorGUI.indentLevel--;
+                if (m_TypeErrorCameras.Any())
                 {
-                    string errorString = "These cameras are not of a valid type:\n";
-                    string validCameras = "";
-                    foreach (var errorCamera in errorCameras)
+                    string message = "";
```

Comment:
Please use a `StringBuilder`

Reply:
Updated. Hoever in theory we do not win much there is a good article https://michaelscodingspot.com/challenging-the-c-stringbuilder/. As most of the time it is gona be 3-4 concetations and we still pay the price of allocation from string builder.